### PR TITLE
Update sharex + aio egg

### PR DESCRIPTION
**Pull request by awaitlol**

### CODE REVIEW

File: egg-a-i-o.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1"
+        "version": "PTDL_v2"
```
Comment (vinc3nz) [inline egg-a-i-o.json:4]: What does this do?

Comment (DEV-DIBSTER) [reply egg-a-i-o.json:4]: My question as well, I'm hoping it's just metadata, but I would need to see.

File: egg-share-x-upload-server.json [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
         "update_url": null
     },
     "exported_at": "2021-05-18T22:02:56-04:00",
     "name": "ShareX Upload Server",
-    "author": "danielpd93@gmail.com",
+    "author": "dan@danbot.host",
     "description": "ShareX Egg, Made by DBH",
     "features": null,
     "images": [
-        "quay.io\/parkervcp\/pterodactyl-images:debian_nodejs-12"
```
Comment (DEV-DIBSTER) [inline egg-share-x-upload-server.json:13]: Are you sure ShareX's server supports the latest version of Node JS?

Comment (ghost) [reply egg-share-x-upload-server.json:13]: I think it would be safer to use NodeJS 16 for the ShareX server

Comment (DEV-DIBSTER) [reply egg-share-x-upload-server.json:13]: I also think it'd be safer to use NodeJS 16, if those changes are made then I can consider this pull request and merge it.

File: egg-a-i-o.json (hunk 1 above)
Comment (awaitlol) [reply egg-a-i-o.json:4]: v1 is exported older then panel version v 1.8 
v2 is exported after v1.8

you can import any v1 egg in to any panel version
v2 can only be imported in panel v 1.8 +

v2 is basically a newer version for eggs

File: egg-share-x-upload-server.json (hunk 2 above)
Comment (awaitlol) [reply egg-share-x-upload-server.json:13]: > I also think it'd be safer to use NodeJS 16, if those changes are made then I can consider this pull request and merge it.

✔ Done

**Review by DEV-DIBSTER — APPROVED** [review]
Looks good, I'm going to merge changes.

**Pull request merged**